frontends/ol-components/src/components/RoutedDrawer/RoutedDrawer.tsx

```diff
@@ -91,25 +79,21 @@ const RoutedDrawer = <K extends string, R extends K = K>(
     }
   }, [router, searchParams, params])
 
+  const childParams = isOpen ? filteredParams : lastValidParams.current
   return (
-    <Drawer
-      open={open}
-      onTransitionExited={removeUrlParams}
-      onClose={setOpen.off}
```

Comment:
In old version, sometimes `onTransitionExited` was not firing, which meant that the query param never got removed from the URL. Consequently, the effect with dependencies `[requiredArePresent, setOpen, requiredParams]` would stop firing, and drawer would stay stuck.